pkgs/tools/z/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, lib, ... }:
+
+stdenv.mkDerivation rec {
+  name = "z-${version}";
+  version = "1.11";
+
+  src = fetchFromGitHub {
+    owner = "rupa";
+    repo = "z";
+    rev = "v${version}";
+    sha256 = "13zbgkj6y0qhvn5jpkrqbd4jjxjr789k228iwma5hjfh1nx7ghyb";
+  };
+
+  buildPhase = " "; # do nothing
+  installPhase = ''
+    install -Dm 644 z.sh $out/share/z.sh
+    gzip z.1
+    install -Dm 644 z.1.gz $out/share/man/man1/z.1.gz
+  '';
```

Comment:
I think this can be written as:

```diff
diff --git a/pkgs/tools/z/default.nix b/pkgs/tools/z/default.nix
index ed5fea65916..97f7ff812b4 100644
--- a/pkgs/tools/z/default.nix
+++ b/pkgs/tools/z/default.nix
@@ -11,11 +11,11 @@ stdenv.mkDerivation rec {
     sha256 = "13zbgkj6y0qhvn5jpkrqbd4jjxjr789k228iwma5hjfh1nx7ghyb";
   };
 
-  buildPhase = " "; # do nothing
+  phases = [ "unpackPhase" "installPhase" ];
+
   installPhase = ''
     install -Dm 644 z.sh $out/share/z.sh
-    gzip z.1
-    install -Dm 644 z.1.gz $out/share/man/man1/z.1.gz
+    install -Dm 644 z.1 $out/share/man/man1/z.1
   '';
 
   meta = {
```